pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -1,146 +1,91 @@
 { lib
-, stdenv
-, python
-, cmake
-, meson
-, vim
-, ruby
-, which
+
+  # nixpkgs functions
+, buildGoModule
+, buildVimPluginFrom2Nix
 , fetchFromGitHub
-, fetchurl
 , fetchpatch
-, llvmPackages
-, rustPlatform
-, buildGoModule
-, pkg-config
-, curl
-, openssl
-, libgit2
-, libiconv
-, xkb-switch
-, fzf
-, skim
-, stylish-haskell
+, fetchurl
+, stdenv
+, substituteAll
+
+  # Language dependencies
+, python
 , python3
+, rustPlatform
+
+  # Misc dependencies
+, Cocoa # vim-addon-manager
+, code-minimap # minimap-vim
+, dasht # vim-dasht
+, direnv # direnv-vim
+, fzf # fzfWrapper
+, gnome3 # vCoolor-vim
+, khard # deoplete-khard
+, languagetool # lens-vim, vim-grammarous
+, llvmPackages # clang_complete, clighter8
+, meson # meson
+, nim # fruzzy
+, nodePackages # coc-*
```

Comment:
that goes to languages.

Reply:
Good old edge cases :p

I think both would be reasonable. I put this here, since it's being used exclusively to get the source for all the `coc` plugins. I'm inclined to leave it here, but I'm happy to move it if you still think is should go in the languages section.